packages/trader/src/Modules/Contract/Components/Sell/sell-info.tsx

```diff
@@ -1,10 +1,14 @@
-import { observer } from 'mobx-react';
```

Comment:
`observer` was deleted as there is no sense of using it without `connect` or `useStore`.